sdk/src/main/kotlin/io/provenance/scope/sdk/SessionBuilder.kt

```diff
@@ -296,44 +296,44 @@ class Session(
 //                .build()
 //        )
 
-        contractSpec.functionSpecsList
-            .filter { it.hasOutputSpec() }
-            .map { it.outputSpec }
-            .map { defSpec ->
-                envelope.contract.considerationsList
-                    .filter { it.hasResult() }
-                    .filter { it.result.output.name == defSpec.spec.name }
-                    .map { it.result.output }
-                    .singleOrNull()
-                    // TODO warn if more than one output with same name.
-                    ?.let {
-                        // Only add the output to the input list if it hasn't been previously defined.
-                        if (builder.inputsList.find { fact -> fact.name == it.name } == null) {
-                            builder.addInputs(
-                                Contracts.Record.newBuilder()
-                                    .setName(it.name)
-                                    .setDataLocation(
-                                        Commons.Location.newBuilder()
-                                            .setClassname(it.classname)
-                                            .setRef(
-                                                Commons.ProvenanceReference.newBuilder()
-                                                    .setHash(it.hash)
-                                                    // TODO where can these be retrieved
-                                                    .setSessionUuid(
-                                                        Util.UUID.newBuilder()
-                                                            .setValueBytes(envelope.ref.sessionUuid.valueBytes).build()
-                                                    )
-                                                    .setScopeUuid(
-                                                        Util.UUID.newBuilder()
-                                                            .setValueBytes(envelope.ref.scopeUuid.valueBytes).build()
-                                                    )
-                                                    .build()
-                                            )
-                                    )
-                            )
-                        }
-                    }
-            }
+//        contractSpec.functionSpecsList
```

Comment:
Why did this get commented out? I remember you mentioning something about it being unused, but I'm not sure I fully understand what it was doing in order to know if it can be removed... seems like it is providing outputs of already executed functions as inputs to other functions, which maybe is a thing that is supposed to stick around? But again, I'm not sure of the usage of that...